Maths/Transpose of a matrix/src/TransposeOfMatrix/MainClass.java

```diff
@@ -0,0 +1,35 @@
+package TransposeOfMatrix;
+
+public class MainClass {
+
+	public static void main(String[] args) {
+		int original[][]={{1,3,4},{2,4,3},{3,4,5}};    
```

Comment:
Please modularise this code by refactoring into a function that accepts an argument and returns the transposed result, so that this function can be used universally.